webpack.functions.js

```diff
@@ -1,6 +1,8 @@
 const webpack = require('webpack')
+const nodeExternals = require('webpack-node-externals')
 
 module.exports = {
+  externals: [nodeExternals()],
```

Comment:
what is this doing?

Reply:
The meetup function relies on several modules which are "backend" and throw errors when we try to bundle them. nodeExternals is the way to deal with this problem, per netlify-lambda (https://github.com/netlify/netlify-lambda#webpack-configuration)